src/stylesheets/functions/_colors.scss

```diff
@@ -0,0 +1,3 @@
+@function color($color: 'base') {
```

Comment:
Unless i'm missing something, there's no "base" color in your map, so this default won't really do anything. Maybe remove the default?

Reply:
Ah, thanks. Copy and pasted from an old project and overlooked this.